Misc/NEWS.d/next/C API/2020-01-07-13-46-40.bpo-39245.G7wog6.rst

```diff
@@ -0,0 +1 @@
+The Vectorcall API (PEP 590) was made public.
```

Comment:
Would you mind to list renamed renamed symbols/macros here? Or is it possible to at least add a link from here to the Doc/c-api/call.rst documentation where the change is documented?